flask/app/variants.py

```diff
@@ -67,7 +66,6 @@ def get_report_df(df: pd.DataFrame, type: str):
                 "variation": "first",
                 "refseq_change": "first",
                 "depth": list,
-                "gene": "first",
```

Comment:
i think it would be helpful to leave the gene in the reports similar to how it was done in the old reports ('long'/tidy style)

Reply:
I think we would have to perform a separate query per gene in the panel since there's no explicit relationship. We could also generate a column in the results in SQL based on which position range matched, but I'm not sure if that's possible through the ORM.